opa fmt panics with future keyword import every, in, and comment

```rego
package test

#
import future.keywords.every

p {
    1 in []
}
```

importing `every` implies importing `in`, it's probably related to that.

`opa fmt f.rego` gives this 💥 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x44c7840]

goroutine 1 [running]:
github.com/open-policy-agent/opa/format.mapImportsToComments(...)
        /Users/stephan/Sources/open-policy-agent/opa/format/format.go:963
github.com/open-policy-agent/opa/format.(*writer).writeImports(0xc000301260, {0xc000700c70, 0x2, 0x2}, {0xc000312318, 0x1, 0xc000311068?})
        /Users/stephan/Sources/open-policy-agent/opa/format/format.go:817 +0x540
github.com/open-policy-agent/opa/format.(*writer).writeModule(0xc000301260, 0x1?)
        /Users/stephan/Sources/open-policy-agent/opa/format/format.go:213 +0x367
github.com/open-policy-agent/opa/format.Ast({0x55fb440?, 0xc000322580?})
        /Users/stephan/Sources/open-policy-agent/opa/format/format.go:99 +0x5dc
github.com/open-policy-agent/opa/format.Source({0x7ff7bfeff777, 0x6}, {0xc00014e200?, 0x0?, 0x7188080?})
        /Users/stephan/Sources/open-policy-agent/opa/format/format.go:30 +0x98
github.com/open-policy-agent/opa/cmd.formatFile(0x6151b68, {0x587d2e0, 0xc00012e008}, {0x7ff7bfeff777, 0x6}, {0x5887d98, 0xc0002225b0}, {0x0?, 0x0?})
        /Users/stephan/Sources/open-policy-agent/opa/cmd/fmt.go:112 +0x285
github.com/open-policy-agent/opa/cmd.opaFmt.func1({0x7ff7bfeff777?, 0x40bad86?}, {0x5887d98?, 0xc0002225b0?}, {0x0?, 0x0?})
        /Users/stephan/Sources/open-policy-agent/opa/cmd/fmt.go:77 +0x5f
path/filepath.walk({0x7ff7bfeff777, 0x6}, {0x5887d98, 0xc0002225b0}, 0x5747788)
        /usr/local/go/src/path/filepath/path.go:418 +0x123
path/filepath.Walk({0x7ff7bfeff777, 0x6}, 0x5747788)
        /usr/local/go/src/path/filepath/path.go:505 +0x6c
github.com/open-policy-agent/opa/cmd.opaFmt({0xc00035aa00?, 0x1, 0x0?})
        /Users/stephan/Sources/open-policy-agent/opa/cmd/fmt.go:76 +0xdf
github.com/open-policy-agent/opa/cmd.glob..func3(0x6102bc0?, {0xc00035aa00?, 0x2?, 0x2?})
        /Users/stephan/Sources/open-policy-agent/opa/cmd/fmt.go:54 +0x27
github.com/spf13/cobra.(*Command).execute(0x6102bc0, {0xc00035a9e0, 0x2, 0x2})
        /Users/stephan/Sources/open-policy-agent/opa/vendor/github.com/spf13/cobra/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x6102940)
        /Users/stephan/Sources/open-policy-agent/opa/vendor/github.com/spf13/cobra/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/stephan/Sources/open-policy-agent/opa/vendor/github.com/spf13/cobra/command.go:902
main.main()
        /Users/stephan/Sources/open-policy-agent/opa/main.go:15 +0x25
```

--- a/format/format.go
+++ b/format/format.go
@@ -60,7 +60,7 @@ func Ast(x interface{}) ([]byte, error) {
 	// or internal.member_3, it will sugarize them into usage of the `in`
 	// operator. It has to ensure that the proper future keyword import is
 	// present.
-	extraFutureKeywordImports := map[string]bool{}
+	extraFutureKeywordImports := map[string]struct{}{}
 
 	// Preprocess the AST. Set any required defaults and calculate
 	// values required for printing the formatted output.
@@ -76,9 +76,9 @@ func Ast(x interface{}) ([]byte, error) {
 		case *ast.Expr:
 			switch {
 			case n.IsCall() && ast.Member.Ref().Equal(n.Operator()) || ast.MemberWithKey.Ref().Equal(n.Operator()):
-				extraFutureKeywordImports["in"] = true
+				extraFutureKeywordImports["in"] = struct{}{}
 			case n.IsEvery():
-				extraFutureKeywordImports["every"] = true
+				extraFutureKeywordImports["every"] = struct{}{}
 			}
 		}
 		if x.Loc() == nil {
@@ -1239,11 +1239,22 @@ func (w *writer) down() {
 
 func ensureFutureKeywordImport(imps []*ast.Import, kw string) []*ast.Import {
 	allKeywords := ast.MustParseTerm("future.keywords")
-	kwPath := ast.MustParseTerm("future.keywords." + kw)
+	kwPath := keyword(kw)
+	every := keyword("every")
 	for _, imp := range imps {
-		if allKeywords.Equal(imp.Path) || imp.Path.Equal(kwPath) {
+		if allKeywords.Equal(imp.Path) ||
+			imp.Path.Equal(kwPath) ||
+			(imp.Path.Equal(every) && kw == "in") { // "every" implies "in", so we don't need to add both
 			return imps
 		}
 	}
-	return append(imps, &ast.Import{Path: kwPath})
+	imp := &ast.Import{
+		Path: kwPath,
+	}
+	imp.Location = defaultLocation(imp)
+	return append(imps, imp)
+}
+
+func keyword(kw string) *ast.Term {
+	return ast.MustParseTerm("future.keywords." + kw)
 }

--- a/format/testfiles/test_in_operator_without_import.rego.formatted
+++ b/format/testfiles/test_in_operator_without_import.rego.formatted
@@ -1,6 +1,7 @@
 package p
 
 import future.keywords.in
+
 import input.foo
 
 r {

--- a/format/testfiles/test_issue_4606.rego
+++ b/format/testfiles/test_issue_4606.rego
@@ -1,0 +1,6 @@
+package p
+
+#
+import future.keywords.every
+
+r { 1 in [] }

--- a/format/testfiles/test_issue_4606.rego.formatted
+++ b/format/testfiles/test_issue_4606.rego.formatted
@@ -1,0 +1,8 @@
+package p
+
+#
+import future.keywords.every
+
+r {
+	1 in []
+}

